js/Game.js

```diff
@@ -6,6 +6,17 @@ class Game  {
     this.playerTurn = 0;
     this.board = null;
     this.currentQuestion;
+    this.dailydouble = {
```

Comment:
Do we need to change these hard coded values?

Reply:
This will be replaced by the function you wrote when it gets merged into master